src/Dashboard/Withdraw/PaymentDetails.tsx

```
@@ -0,0 +1,154 @@
+    settings,
+}: {
+    bodyData: UseBalanceReturn;
+    masterLoading: boolean;
+    withdrawRequests: UseWithdrawRequestsReturn;
+    settings: UseWithdrawSettingsReturn;
+} ) {
+    const navigate = useNavigate();
+
+    if (
+        ! bodyData ||
+        ! bodyData.hasOwnProperty( 'isLoading' ) ||
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)